Makefile

```diff
@@ -36,7 +41,10 @@ src/dashboard:
 src/globus_handler:
 	git clone https://github.com/whole-tale/globus_handler src/globus_handler
 
-sources: src/gwvolman src/wholetale src/wt_data_manager src/wt_home_dir src/dashboard src/globus_handler src/girderfs
+src/dashboard_next:
```

Comment:
Makefile rule in this case should match the name of the directory it creates (`src/wt_ng_dashboard`). The git repo is called `ngx-dashboard`, can we unify them?